src/preferencesdialog.ui

```diff
@@ -201,6 +201,13 @@
          </property>
         </widget>
        </item>
+       <item row="7" column="0">
+        <widget class="QCheckBox" name="useBlurOnZoomBox">
+         <property name="text">
+          <string>Smooth images on zoom</string>
```

Comment:
You've chosen a good name for the option ("Smooth images on zooming" — an "ing" was missing) because it isn't about blurring. Please change the key, and method names accordingly (e.g., `SmoothOnZoom` instead of `UseBlurOnZoom`, `smoothOnZoom()` instead of `useBlurOnZoom()`, etc.)